index.html

```diff
@@ -22,7 +22,7 @@ <h1>Today's To Do</h1>
         </form>
         <hr>
         <ul id="to-do-list">
-            <!--TASKS ELEMENTS DINAMICALLY INSERTED WITH JAVASCRIPT-->
+            <!-- TASKS ELEMENTS DINAMICALLY INSERTED WITH JAVASCRIPT -->
         </ul>
         <hr>
         <button type="button" class="clear-btn">Clear all completed</button>
```

Comment:
- I suggest you use apply Html semantics here. You can use a header for the heading instead of a div and also the forms can be in a section because Html semantics makes your code more comprehensible by better defining the different sections and layout of web pages. Great Job by the way